docs/Compile.rst

```diff
@@ -469,7 +469,8 @@ Dependencies
 ------------
 You will need the following:
 
-* Microsoft Visual C++ 2015 or 2017
+* Microsoft Visual C++ 2022, 2019, 2017, or 2015 (optional)
```

Comment:
Instead of "optional", how about "if editing code"?

Reply:
it's a perfectly reasonable option to use VS Code to edit code and let VS Code call the build tools. the integration isn't even all that bad.

Reply:
Ah, ok. I wasn't aware that VS Code and Visual C++ were different products. The last time I did Windows programming, there was just Visual Studio, and Visual C++ just plugged into that.